### PR TITLE
Remove unused session variables

### PR DESCRIPTION
**Pull request by EmmyMiao87**
Remove unused session variables named sql_select_limit.

Change-Id: I35244cd6e490638d08f86a5c08131ff3cdba80b6

### CODE REVIEW

File: fe/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -556,7 +548,8 @@ public void readFields(DataInput in) throws IOException {
             queryCacheType = in.readInt();
             autoIncrementIncrement = in.readInt();
             maxAllowedPacket = in.readInt();
-            sqlSelectLimit = in.readLong();
```
Comment (wuyunfeng) [inline fe/src/main/java/org/apache/doris/qe/SessionVariable.java:559]: Maybe you should consider BWC.

Comment (morningman) [reply fe/src/main/java/org/apache/doris/qe/SessionVariable.java:559]: What is BWC?

**Review by morningman — COMMENTED** [review]
This variable is used for compatibility for some low version mysql lib.
Better not to remove it.

File: fe/src/main/java/org/apache/doris/qe/SessionVariable.java (hunk 1 above)
Comment (wuyunfeng) [reply fe/src/main/java/org/apache/doris/qe/SessionVariable.java:559]: BackWardsCompatible

**Comment by morningman** [comment]
Should not remove.

**Pull request closed without merging**